Task 4/script.js

```diff
@@ -8,4 +8,36 @@ Pastaba: Sukurta kortelė, kurioje yra informacija apie automobilį (brand), tur
 būti stilizuota su CSS ir būti responsive;
 -------------------------------------------------------------------------- */
 
-const ENDPOINT = 'cars.json';
+document.addEventListener('DOMContentLoaded', async () => {
+  const ENDPOINT = 'cars.json';
+  const renderData = (brands) => {
+    const outputContainer = document.getElementById('output');
+    outputContainer.innerHTML = '';
+    brands.forEach((carData) => {
+      const car = document.createElement('div');
+      car.classList.add('car');
+      const brandName = document.createElement('span');
+      brandName.classList.add('brand-name');
+      brandName.textContent = carData.brand;
+      const modelsName = document.createElement('span');
+      modelsName.textContent = carData.models;
+      modelsName.classList.add('models-list');
+      car.append(brandName);
+      car.append(modelsName);
+      outputContainer.append(car);
+    });
+  };
+
+  const brands = async () => {
+    try {
+      const response = await fetch(ENDPOINT);
+      const data = await response.json();
```

Comment:
By default json join() jungia per kablelį be jokių tarpų. Jums reikėjo paduoti parametrą kaip ", ", tada būtų tvarkingiau. Manau dėl to ir responsyvumas gan minimalus, nes daugeliu atvejų markės yra vienas milžiniškas žodis, kuris nelūžta.